recipes/CIS_2012r2_L1.rb

```diff
@@ -0,0 +1,571 @@
+#
+# Cookbook Name:: base-win2012-hardening
+# Recipe:: CIS_2012r2_L1
+#
+# Copyright (c) 2017 Matt Tunny, All Rights Reserved.
+#
+# Setting below break test-kitchen but required in production, Also this recipe does not include firewall settings.
+# unless ENV['TEST_KITCHEN']
+
+# NTLM Hardening -- This settings breaks WinRM
+if node['NTLM_Harden'] == true
+  # System Policys
+  registry_key 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Policies\System' do
+    values [{ name: 'LocalAccountTokenFilterPolicy', type: :dword, data: 0 }] # This breaks test-kitchen if enabled
+    action :create
+  end
+  # NTLM Hardening
+  registry_key 'HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Lsa\MSV1_0' do
+    values [{ name: 'RestrictReceivingNTLMTraffic', type: :dword, data: 2 },
+            { name: 'RestrictSendingNTLMTraffic', type: :dword, data: 2 }]
+    action :create
+  end
+end
+
+# Winlogon Settings
```

Comment:
I am thinking about different recipes for each component, like `winlogon.rb`, `lsa.rb` what do you think?